docs/narr/introduction.rst

```diff
@@ -73,44 +73,47 @@ working on a framework that is up-to-date and forward-looking.
 Tested
 ~~~~~~
 
-Untested code is broken by design. The Pyramid community has a strong testing
-culture and our framework reflects that. Every release of Pyramid has 100%
-statement coverage [#]_ and 95% decision/condition coverage. [#]_ It is
-automatically tested using `Travis <https://travis-ci.org/Pylons/pyramid>`_ and
-`Jenkins <http://jenkins.pylonsproject.org/job/pyramid/>`_ on Python 2.7,
-Python 3.4, Python 3.5, and PyPy after each commit to its GitHub repository.
+Untested code is broken by design.
+The Pyramid community has a strong testing culture and our framework reflects that.
+Every release of Pyramid has 100% statement coverage (as measured by `coverage <https://coverage.readthedocs.io>`_)
+and 95% decision/condition coverage. (as measured by `instrumental <http://instrumental.readthedocs.io/en/latest/intro.html>`_)
+It is automatically tested using `Travis <https://travis-ci.org/Pylons/pyramid>`_
+and `Jenkins <http://jenkins.pylonsproject.org/job/pyramid/>`_
+on supported versions of Python after each commit to its GitHub repository.
```

Comment:
Although Plone docs use semantic line breaks, that went over like a lead balloon with Pylons Project core developers: https://github.com/Pylons/pyramid/issues/2633

One line per sentence was kinda sorta OK.

One line per paragraph in .rst only is preferred.

Otherwise the changes look good.